packages/components/src/components/DiagramSequence.vue

```diff
@@ -84,7 +83,7 @@ export default {
         const specification = Specification.build(appMapObj, { loops: true });
         result = buildDiagram('<an AppMap file>', appMapObj, specification);
       }
-      assert(result);
+      if (!result) throw Error;
```

Comment:
These should all be `throw new Error()` otherwise the stack won't be reported

Reply:
Okay, good to know! I fixed this and rebased the branch.